generative_model_training/05_plot_labels_sampling_performance.py

```diff
@@ -0,0 +1,511 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:snorkeling_full_text]
+#     language: python
+#     name: conda-env-snorkeling_full_text-py
+# ---
+
+# # Plot Label Resampling Performance
+
+# +
+from pathlib import Path
+
+import scipy.stats
+import pandas as pd
+import plotnine as p9
+import plydata as ply
+# -
+
+performance_dfs = pd.concat(
+    [
+        pd.read_csv(str(df_path), sep="\t")
+        >> ply.define(lf_num=ply.expressions.if_else("lf_num > 16", 99, "lf_num"))
```

Comment:
It may be helpful to explain what `lf_num` is at the top since a lot of your code references it. Is it the number of label functions used? Or is it a number referencing a specific label function?

Reply:
>It may be helpful to explain what lf_num is at the top since a lot of your code references it. Is it the number of label functions used? Or is it a number referencing a specific label function?

Yes. It is the number of label functions that were sampled for this experiment. I'll add a description at the top to disambiguate this variable name.